internal/cmd/skupper/common/flags.go

```diff
@@ -70,6 +70,10 @@ for other Kubernetes flavors, loadbalancer is the default.`
 	FlagDescListenerPort = "The port of the local listener"
 	FlagNameListenerHost = "host"
 	FlagDescListenerHost = "The hostname or IP address of the local listener. Clients at this site use the listener host and port to establish connections to the remote service."
+
+	FlagNameWait       = "wait"
+	FlagDescWait       = "Wait for the given status before exiting. Choices: pending, configured, ready"
```

Comment:
As per the latest discussion, I believe we need to remove "pending" from the description.

Reply:
Replaced by `none`, as discussed.